### PR TITLE
Add dependabot configuration

### PR DESCRIPTION
**Pull request by yeikel**
See https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github_actions"
```
Comment (dondonz) [inline .github/dependabot.yml:7]:
```suggestion
  - package-ecosystem: "github-actions"
```
Could you please change it to a dash? From the documentation: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

**Review by dondonz — CHANGES_REQUESTED** [review]
Great idea to add Dependabot, thanks

File: .github/dependabot.yml (hunk 1 above)
Comment (yeikel) [reply .github/dependabot.yml]: Thank you for the catch. I confused them because that's the name internally. Thanks for the review!

**Review by dondonz — APPROVED** [review]
Thanks so much for adding Dependabot!

And thanks for your first PR on this repo (hopefully, first of many to come!)

**Pull request merged**